### PR TITLE
Add statsd metricbeat input.

### PR DESCRIPTION
**Pull request by lalit-satapathy**
## What does this PR do?

Add **statsd/metrics** missing input. We are working on **integrations/input package** using **statsd/metrics** In both the cases we noticed:

- Some issues in the new agent, where **statsd** metrics were not pushed to ES.
- Working for older agents but not in new agents.
-  statsd entry to be missing in the recently added metricbeat spec file in V2 elastic-agent.

CC: @ishleenk17 

## Why is it important?

- Gating the development of new statsd input package, while using latest V2 elastic-agent.

## Checklist

NA

-->

- [X] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

### CODE REVIEW

File: specs/metricbeat.spec.yml [hunk 1]
```diff
@@ -287,6 +287,13 @@ inputs:
     shippers: *shippers
     command:
       args: *args
+  - name: statsd/metrics
+    description: "Statsd metrics"
+    platforms: *platforms
+    outputs: *outputs
+    shippers: *shippers
+    command:
+      args: *args
```
Comment (cmacknz) [inline specs/metricbeat.spec.yml:296]:
```suggestion
    command: *command
```

This is the format you want to keep when the conflict is resolved.

Comment (lalit-satapathy) [reply specs/metricbeat.spec.yml]: done

**Comment by lalit-satapathy** [comment]
Hi @cmacknz,

We may have few more metricbeat inputs missing. Example - _memcached_. How do we ensure all inputs used in integrations are covered in the spec file?

**Comment by cmacknz** [comment]
> How do we ensure all inputs used in integrations are covered in the spec file?

An automated test that configures the necessary input would be the best way, if there is an integration that requires that input then a test in https://github.com/elastic/integrations/ would catch it.

Otherwise manual review of the spec file is the other choice. We hope to eventually move the Beat spec files in the Beats repository so that it is easier to update this list with one change.

**Review by cmacknz — APPROVED** [review]

**Comment by lalit-satapathy** [comment]
> > How do we ensure all inputs used in integrations are covered in the spec file?
> 
> An automated test that configures the necessary input would be the best way, if there is an integration that requires that input then a test in https://github.com/elastic/integrations/ would catch it.
> 
> Otherwise manual review of the spec file is the other choice. We hope to eventually move the Beat spec files in the Beats repository so that it is easier to update this list with one change.

Hi Craig,
Wondering if the agent upgrades are running the individual integration system tests?  We can do manual review otherwise. Currently only memchached seems to be missing AFAIK.

**Comment by cmacknz** [comment]
> Wondering if the agent upgrades are running the individual integration system tests?

The agent build doesn't trigger the tests in the https://github.com/elastic/integrations/ repository. We do look at them and help triage when the agent is potentially at fault.

From what I can tell there isn't a failing memcached test in the last test run that included all integrations https://fleet-ci.elastic.co/job/ingest-manager/job/integrations/job/main/648/#showFailuresLink to have caught this.

**Pull request merged**